src/assets/data/team.json

```diff
@@ -2,69 +2,69 @@
   {
     "img": "/src/assets/img/team/Benjamin Frost.webp",
     "name": "Benjamin Frost",
-    "lastRole": "Jury",
+    "lastRole": { "de": "Jury", "en": "Jury" },
     "hasMultipleRoles": true
   },
   {
     "img": "/src/assets/img/team/Jonas Wanke.webp",
     "name": "Jonas Wanke",
-    "lastRole": "Expert",
+    "lastRole": { "en": "Expert", "de": "Experte" },
     "skill": "Skill 08",
     "hasMultipleRoles": true
   },
   {
     "img": "/src/assets/img/team/Dr. Olaf Kappler.webp",
     "name": "Dr. Olaf Kappler",
-    "lastRole": "Expert",
+    "lastRole": { "en": "Expert", "de": "Experte" },
     "skill": "Skill 09"
   },
   {
     "img": "/src/assets/img/team/Doreen Kappler.webp",
     "name": "Doreen Kappler",
-    "lastRole": "Good Fairy"
+    "lastRole": { "de": "Good Fairy", "en": "Good Fairy" }
```

Comment:
```suggestion
    "lastRole": { "de": "Good Fairy", "en": "Gute Fee" }
```

Reply:
"Good Fairy" was the original German (i.e. on our PowerPoints we use it as well). Are we sure about this change? I do not mind either way, I just think the English version sounds better